go-controller/pkg/ovn/controller/services/load_balancer.go

```diff
@@ -32,6 +43,8 @@ type lbConfig struct {
 	// that means, skipSNAT, and remove any non-local endpoints.
 	// (see below)
 	externalTrafficLocal bool
+	// either 1) clusterIP or 2) nodePort or 3) LoadBalancerOrExternalIP (with ingress and/or externalIPs)
+	svcType ServiceType
```

Comment:
this could just be hasNodePort bool. Maybe I previously steered you wrong here. i thought this value was used elsewhere in the code to make decisions, but it looks like its only used in code you added? 
```if config.externalTrafficLocal && config.svcType == ServiceTypeNodePort {```

Reply:
correct, I added it specifically for that use case alone. I can rewind it.